test/built-ins/Proxy/has/call-hasownproperty.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.7
+description: >
+    Object.prototype.hasOwnProperty triggers trap.call(handler, target, P);
+info: >
+    [[HasProperty]] (P)
+
+    ...
+    2. Let handler be the value of the [[ProxyHandler]] internal slot of O.
+    ...
+    5. Let target be the value of the [[ProxyTarget]] internal slot of O.
+    6. Let trap be GetMethod(handler, "has").
+    ...
+    9. Let booleanTrapResult be ToBoolean(Call(trap, handler, «target, P»)).
+    ...
+---*/
+
+var target = Object.create(Array.prototype);
```

Comment:
`target` can be defined via a simple object literal (no need to have ArrayPrototype in its prototype chain)